app/components/Card/card.css

```diff
@@ -0,0 +1,55 @@
+.card {
+  height: 328px;
+  padding: var(--spacing-24);
+  display: flex;
+  background-color: var(--colors-cool-grey-100);
+  border-radius: var(--border-radius);
+  box-shadow: 0px var(--spacing-16) var(--spacing-40) rgba(175, 183, 194, 0.2);
+}
+
+.card-content {
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  padding: var(--spacing-32);
+}
+
+.card-icon {
+  height: 100%;
+  background-color: var(--colors-teal-50);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: var(--border-radius);
+  width: 30%;
+  flex-shrink: 0;
+}
+
+.card-impact {
+  color: var(--colors-teal-500);
+  height: fit-content;
+  width: fit-content;
+  padding-left: 10px;
+  padding-top: 2px;
+  padding-bottom: 2px;
+  padding-right: 10px;
```

Comment:
Bryce and I are flagging this heh. Padding values should pretty much always be 4px, 8px, 16px, etc. (variable)